lib/common/components/MapModal.js

```diff
@@ -36,33 +63,33 @@ export default class MapModal extends Component {
     return this.refs.map
   }
 
-  _onEditPath (e) {
+  _onEditPath () {
     console.log('Path edited !')
   }
 
-  _onCreate (e) {
-    polyline = e.layer
-    this.setState({rectangle: e.layer})
-    // To edit this polyline call : polyline.handler.enable()
-    console.log('Path created !', polyline)
+  // _onCreate (e) {
```

Comment:
If we still need this even with comments, please indicate why and perhaps create a github issue.